code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -503,10 +500,18 @@
 
 /datum/reagent/medicine/ephedrine/on_mob_life(mob/living/M)
 	M.status_flags |= GOTTAGOFAST
```

Comment:
This makes sense, considering it was removed from meth which keeps stamina regen, and this loses stamina regen.